AdminRouter/start-windows-build.ps1

```diff
@@ -0,0 +1,266 @@
+Param(
+    [Parameter(Mandatory=$false)]
+    [string]$GitURL="https://github.com/apache/httpd", 
+    [Parameter(Mandatory=$false)]
+    [string]$Branch="master",
```

Comment:
Looking at https://github.com/apache/httpd, I see that their main branch is `trunk` and not `master`.

I would suggest the default parameter value matching their main branch.

Reply:
True that. Basically, this `AdminRouter\start-windows-build.ps1` script it's not building anything. It is just taking the upstream zip package and re-package it, right ?

Do we really need to this ? Can't we use the upstream zip package directly in the setup script ?

Reply:
As you pointing out we are not building anything, it's more about customizing the config for setting up the http server.  Given the limited feature we need from the Apache server, we probably won't have a need to generate a customized build for DCOS.

The pkgpanda will have to convert the logics here to setup the same server config.